staging/src/k8s.io/kube-scheduler/config/v1alpha1/types.go

```diff
@@ -127,3 +136,64 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string `json:"lockObjectName"`
 }
+
+// Plugins include multiple extension points. When specified, the list of plugins for
+// a particular extension point are the only ones enabled. If an extension point is
+// omitted from the config, then the default set of plugins is used for that extension point.
```

Comment:
where is the default set of plugins specified?

Reply:
We currently do not have any plugins and therefore, no set of default plugins. We will add plugins in the next step and after the main components and extension points of the framework are built.